packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -63,6 +72,18 @@ export async function ensureExtensionsIds(
     remoteExtensions = remoteExtensions.concat(newRemoteExtensions)
   }
 
+  if (marketingActivityExtensionsToMigrate.length > 0) {
+    const confirmedMigration = await extensionMigrationPrompt(marketingActivityExtensionsToMigrate, false)
+    if (!confirmedMigration) throw new AbortSilentError()
+    const newRemoteExtensions = await migrateMarketingActivityExtensions(
```

Comment:
same here, you should be able to use the generic `migrateAppModules`, as payments do just above this